src/Dashboard/container.js

```diff
@@ -0,0 +1,27 @@
+// 3rd-party imports
```

Comment:
Container should be for `redux` containers, sorry 🙏  It should be `src/Dashboard/component.js`